backend/api/Program.cs

```diff
@@ -24,10 +25,13 @@ public static void Main(string[] args)
         /// </summary>
         /// <param name="args"></param>
         /// <returns></returns>
-        public static IWebHostBuilder CreateWebHostBuilder(string[] args) =>
-            WebHost.CreateDefaultBuilder(args)
-            .UseContentRoot(Directory.GetCurrentDirectory())
-            .UseUrls("http://*:8080")
-            .UseStartup<Startup>();
+        public static IWebHostBuilder CreateWebHostBuilder(string[] args)
+        {
+            DotNetEnv.Env.Load();
+            return WebHost.CreateDefaultBuilder(args)
+                .UseContentRoot(Directory.GetCurrentDirectory())
+                .UseUrls(Environment.GetEnvironmentVariable("ASPNETCORE_URLS"))
```

Comment:
You will need to update your `.env` file to have `ASPNETCORE_URLS=http://*:8080`.  If we don't do this it won't run locally for debugging.

Reply:
When debugging you need to update your `.env` to this `ASPNETCORE_URLS=http://*:5000`

Reply:
I thought NET Core already set the urls based on that env var....

Reply:
They do, but when you run locally that won't use the `.env` file to initialize the environment variables.